### PR TITLE
Development book tests

### PR DESCRIPTION
**Pull request by Alexander1iubchenko**

### CODE REVIEW

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
too many changes unrelated to this homework. Please update the PR so that it is reviewable

File: src/test/java/com/example/application/controller/BookControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                        .content(jsonRequest)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isCreated())
+                .andReturn();
+
+        BookDto expected = new BookDto()
+                .setTitle(bookRequestDto.getTitle())
+                .setAuthor(bookRequestDto.getAuthor())
+                .setPrice(bookRequestDto.getPrice())
+                .setIsbn(bookRequestDto.getIsbn());
```
Comment (d1sam) [inline src/test/java/com/example/application/controller/BookControllerTest.java]: you can use as expected book your `bookRequestDto` fields

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but please look at my minor comment and make one change!

**Review by yevheniisynytsia — APPROVED** [review]
🔥

**Pull request merged**